packages/core/test/baseTransfer.test.ts

```diff
@@ -0,0 +1,73 @@
+import { enableFetchMocks } from 'jest-fetch-mock';
+import { BaseTransfer, BaseTransferParams } from '../src/baseTransfer.js';
+import { ConfigUrl } from '../src/constants.js';
+import { Config, Environment, EvmResource, ResourceType } from '../src/index.js';
+import { mockedDevnetConfig } from './constants.js';
+
+enableFetchMocks();
+
+const TRANSFER_PARAMS: BaseTransferParams = {
+  source: 111,
+  destination: 111,
+  resource: '0x0000000000000000000000000000000000000000000000000000000000000000',
+  sourceAddress: '0x00',
+};
+
+class Transfer extends BaseTransfer {
+  constructor(params: BaseTransferParams, config: Config) {
+    super(params, config);
+  }
+}
+
+describe('BaseTransfer', () => {
+  let config: Config;
+
+  beforeAll(async () => {
+    jest.clearAllMocks();
+    fetchMock.resetMocks();
+    fetchMock.doMock();
+    fetchMock.mockIf(ConfigUrl.DEVNET.toString(), JSON.stringify(mockedDevnetConfig));
+    config = new Config();
+    await config.init(Environment.DEVNET);
+  });
+
+  it('should be able to instantiate a transfer object', async () => {
+    const transfer = new Transfer(TRANSFER_PARAMS, config);
+    expect(transfer).toBeTruthy();
+    expect(transfer).toBeInstanceOf(Transfer);
```

Comment:
if this is true, do we need to one above?

Reply:
nah its redundant i have removed